doc/source/ray-core/fault_tolerance/actors.rst

```diff
@@ -135,6 +135,38 @@ original handle to the actor.
 
 If ``max_restarts`` is set, you can also allow Ray to automatically restart the actor by passing ``no_restart=False`` to ``ray.kill``.
 
+Unavailable Actors
+----------------------
+
+When an actor can't accept method calls,
+a method call or a `ray.get` on the method return object reference may raise
+`ActorUnavailableError`. This exception indicates the actor isn't accessible for the
+moment, but may recover after some wait and retry. Typical cases include:
+
+- The actor is starting, for example it's still running the class constructor.
+- The actor is restarting, for example it's waiting for resources.
+- The actor is experiencing transient network issues, for example connection breaks.
+- The actor is dead but the local Raylet hasn't yet reported to the GCS.
+
+When a `actor.method.remote()` or `ray.get()` call raises this exception, there is no guarantee on
+whether the actor received the invocation or not. If the method has side effects, it may or may not
+be observable after the actor recovered. However Ray do guarantee at-most-once, that a single invocation won't land on the actor twice, unless the actor or the method is configured with retries described in the next section.
+
+The actor may or may not recover in the next calls. Those subsequent calls
```

Comment:
If the task has retries available and the retries happen while the actor is recovering, do the retried tasks get queued or do they fail with ActorUnavailableError?

If the latter, how do we know how to set max_retries?

Reply:
Added a unit test `test_task_retries`